workflows/diagnostics/fv3net/diagnostics/prognostic_run/compute.py

```diff
@@ -526,6 +535,40 @@ def compute_hist_2d_bias(diag_arg: DiagArg):
     return _assign_diagnostic_time_attrs(hist2d_prog, diag_arg.prediction)
 
 
+@registry_3d.register("300_700_time_dependent_zonal_mean")
```

Comment:
If going to the trouble to make compute these (time, lat) fields, can we see them as Hovmullers?

Reply:
Good idea.